aspnet/fundamentals/owin/sample/src/OwinSample/Startup.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNet.Builder;
```

Comment:
Yes, they're using the Entropy samples with minimal modification. I wasn't sure if the Entropy samples were baked, so since these were each very small samples, I opted to just include them with the article's samples (which also allows literalinclude to work for dispalying the code).